add-ons/grafana/README.md

```diff
@@ -0,0 +1,41 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon.
+
+Verify you have installed these addons, by following the [general readme](../../Readme.adoc#download-and-use-community-add-ons).
```

Comment:
Links is broken. It `Readme.adoc` should be `README.adoc`

Reply:
Removed this line to avoid misunderstanding.